conf/zoo_sample.cfg

```diff
@@ -6,6 +6,10 @@ initLimit=10
 # The number of ticks that can pass between 
 # sending a request and getting an acknowledgement
 syncLimit=5
+# enable TTL Nodes
+# IMPORTANT: when enabled, your server ID cannot be greater than 254
+# See https://zookeeper.apache.org/doc/r3.5.3-beta/zookeeperAdmin.html#sc_configuration for details
```

Comment:
Perhaps note the document and section, rather than the link? It's going to get out of date quickly.